js/message_test.js

```diff
@@ -184,14 +180,7 @@ describe('Message test suite', function() {
     var response = new proto.jspb.test.DefaultValues();
 
     // Test toObject
-    var expectedObject = {
-      stringField: defaultString,
-      boolField: true,
-      intField: 11,
-      enumField: 13,
-      emptyField: '',
-      bytesField: 'bW9v'
-    };
+    var expectedObject = {};
```

Comment:
The existing semantics are different for fields with and without explicit defaults. I assume you mean the former? Making sure because some assertions will need to be changed either way.